dll/steam_remote_storage.cpp

```diff
@@ -497,7 +496,7 @@ bool Steam_Remote_Storage::GetUGCDetails( UGCHandle_t hContent, AppId_t *pnAppID
 {
     PRINT_DEBUG_ENTRY();
     std::lock_guard<std::recursive_mutex> lock(global_mutex);
-    
+
```

Comment:
on my end I don't see the changes nor git detects it since I use LF for everything, can't fix, don't even know what the original line ending was
inside this mess I removed 2 duplicate lines though.

Reply:
oh wait I'm an idiot, one moment

Reply:
yeah fix already, check above comment